LoveMyCapibara/LoveMyCapibara/Views/Scenes/Settings/Subviews/ThemeSelector.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ThemeSelector.swift
+//  LoveMyCapibara
+//
+//  Created by userext on 01/06/23.
+//
+
+import SwiftUI
+
+
+struct ThemeSelector: View {
+    @SceneStorage("preferredColorScheme") var selectedOption = 1
+    @Environment(\.colorScheme) var colorScheme
+    
+    var theme: ColorScheme {
```

Comment:
Testa se consegue transformar ela em um Lazy var ou um State :D assim quando for modificado, é modificado junto talvez